docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -2,33 +2,33 @@
 
 = Performing a fast-forward upgrade on Compute services
 
-Compute services rolling upgrade cannot be done during adoption,
-there is in a lock-step with Compute control plane services, because those are managed independently by data plane ansible and Kubernetes Operators.
-The Compute service operator and OpenStack Operator ensure upgrading
-is done independently of each other, by configuring
-`[upgrade_levels]compute=auto` for Compute services. Compute control plane
-services apply the change right after custom resource (CR) is patched. Compute data plane services will catch up the same config change with ansible deployment later on.
+You must upgrade the Compute services from {rhos_prev_long} {rhos_prev_ver} to {rhos_long} {rhos_curr_ver} on the control plane and data plane. The upgrade procedure involves the following tasks:
 
-//[NOTE]
-//Additional orchestration happening around the FFU workarounds
-//configuration for Compute data plane service is a subject of future changes. kgilliga: We don't comment on future changes downstream.
+* Update the cell1 Compute data plane services version.
+* Remove pre-fast-forward upgrade workarounds from the Compute control plane services and Compute data plane services.
+* Run Compute database online migrations to update live data.
 
 .Procedure
 
-. Wait for cell1 Compute data plane services version updated (it may take some time):
+. Wait for cell1 Compute data plane services version to be updated:
 +
 ----
-oc exec openstack-cell1-galera-0 -c galera -- mysql -rs -uroot -p$PODIFIED_DB_ROOT_PASSWORD \
+$ oc exec openstack-cell1-galera-0 -c galera -- mysql -rs -uroot -p$PODIFIED_DB_ROOT_PASSWORD \
     -e "select a.version from nova_cell1.services a join nova_cell1.services b where a.version!=b.version and a.binary='nova-compute';"
 ----
 +
-The above query should return an empty result as a completion criterion.
+[NOTE]
+====
+The query returns an empty result when the update is completed. No downtime is expected for virtual machine workloads.
```

Comment:
"is expected" is passive.
You drop "No downtime is expected for virtual machine workloads." entirely - it also sounds a bit like a promise or marketing talk.